webpack/scenes/Subscriptions/SubscriptionReducer.js

```diff
@@ -50,7 +49,6 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case SUBSCRIPTIONS_REQUEST:
     case UPDATE_QUANTITY_REQUEST:
-    case DELETE_SUBSCRIPTIONS_REQUEST:
```

Comment:
Also in this file is:    

```
case DELETE_SUBSCRIPTIONS_SUCCESS:
      return state.set('loading', false);
```

I *think* this is no longer required

Reply:
Yes. we don't need to set `loading` to `false` anymore. I am now using `DELETE_SUBSCRIPTIONS_SUCCESS` to set `deleteButtonDisabled` to `true`.

Reply:
Seems like we need to set loading to false because loading is set to true in the initial state. A test case failed. So I added this - `state.set('loading', false) `  back.